### PR TITLE
[DOCS] Document Linux OS support for host isolation

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1013, #831, #1052.

Note: This pull request focuses only on listing the specific Linux distributions supported for host isolation. Other updates to the Host isolation topic (some UX updates) are in PR #[1010: [DOCS] Manage Endpoint - UX Updates](https://github.com/elastic/security-docs/pull/1010).

Preview [here](https://security-docs_1054.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-ov.html).

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
LGTM!

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 1]
```diff
@@ -4,7 +4,17 @@
 
 Host isolation allows you to isolate hosts from your network, blocking communication with other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. Isolated hosts, however, can still send data to {es} and {kib}.
 
-NOTE: For {stack} version >= 7.14.0, host isolation is supported for Windows and macOS endpoints only. To isolate and release hosts, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+[NOTE] 
+=========================
+For {stack} version >= 7.15.0, host isolation is supported for endpoints in Windows, macOS, and these Linux distributions:
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:9]:
```suggestion
For {stack} version >= 7.15.0, host isolation is supported for endpoints running Windows, macOS, and these Linux distributions:
```

**Review by jmikell821 — APPROVED** [review]
Super small nit - looks good otherwise.

**Review by jmikell821 — APPROVED** [review]
Looks good to merge/backport!

**Pull request merged**